addons/io_hubs_addon/io/gltf_exporter.py

```diff
@@ -37,7 +42,7 @@ def export_callback(callback_method, export_settings):
     # mid iteration and so multiple callbacks could be executed for the same
     # component/host.
 
-    for scene in bpy.data.scenes[:]:
+    for scene in bpy.data.scenes[:]:  # I don't think we need to copy the dicts [:], since we can't even alter the list, this would save mem
```

Comment:
The explanation for this list copy is in the preview comment block. If that's not true anymore, we should remove that comment block, otherwise we should remove this one. @Exairnous

Reply:
As far as I know, the preview comment block is still valid.  Also, given the recent issue I found with data corruption when components were removed from the original list but still present in the copy, we should maybe think about putting in some extra guards at some point.

Reply:
Blender scenes, objects, data can only be modified with bpy.ops.
bpy.data.objects for example by delete or add operators. I don't think you want objects that were previously deleted in the list and I think adding objects should also add to that list. However the copy is discarded as soon as the export_callback is complete, so I don't think it matters at this point. 
"when components were removed from the original list but still present in the copy," is there a reason why this should be possible at all?

Reply:
I think we discussed this at a dev meetup, but I'm putting this here for the record.  My experience has been that Blender data can be modified outside of bpy.ops and that anything added or removed during this loop you probably don't want to operate on within it (you want to stay with the original items unless they've been removed).  As for removing, it's probably unlikely here, but the callbacks can pretty much do anything they want, so it's not out of the question, e.g. perhaps something gets added pre-export and removed post-export.